.github/workflows/ci_release.yml

```diff
@@ -29,9 +29,13 @@ jobs:
 
   lint:
     uses: ./.github/workflows/lint.yml
+    with:
+      GO_VERSION: "1.21.5"
 
   test:
     uses: ./.github/workflows/test.yml
+    with:
+      GO_VERSION: "1.21.5"
```

Comment:
This is causing the workflow not to run. GO_VERSION isn't defined as an input because the files were updated to use the go.mod file. 

I'd remove and it should be fine.